app/assets/javascripts/modules/media_viewer.js

```diff
@@ -146,11 +147,22 @@
       return hlsSource.prop('src').indexOf('/mp3:') >= 0;
     }
 
+    function ms10Browser() {
+      var fingerprinter = new ClientJS();
```

Comment:
I'm pulling in Client.js, so it *is* in fact defined. Let me know if there's a way to make Hound quiet.